samples/toolbar/src/main/res/drawable/toggle_background.xml

```diff
@@ -6,6 +6,12 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:color="@color/photonGrey90">
 
+    <item android:id="@android:id/mask">
```

Comment:
@mcomella this approach fix the issue for your use case?

Reply:
I'm not sure – I was specifically trying to address the problem that the ripple appears on non-toggle ToolbarAction but only appears in the enabled -> disabled state on toggleable actions: https://drive.google.com/file/d/1fS_A4HtXQFB6AT_StketLG-gpRf89d7_/view?usp=sharing

However, I didn't realize that some part of this is done in the application – we set the ripple on our elements – but I don't know why the ripple effect shouldn't just work going disable -> enabled (maybe because there needs to be a background color for that to work?). I'm not sure if this is a bug in the product or in components.